src/IB/IBFEMethod.cpp

```diff
@@ -2758,6 +3402,8 @@ IBFEMethod::spreadTransmissionForceDensity(const int f_data_idx,
     fe.evalQuadraturePointsFace();
     fe.evalQuadratureWeightsFace();
     const size_t X_sys_idx = fe.registerInterpolatedSystem(X_system, vars, vars, &X_ghost_vec);
+    const size_t Phi_sys_idx = Phi_vec ? fe.registerInterpolatedSystem(*Phi_system, Phi_vars, no_vars, Phi_vec) :
```

Comment:
I think we now have an `invalid_index` that you could use here.

Reply:
Whoops, that is for SAMRAI indices, not libMesh.

Is there a libMesh invalid index variable? I am not sure...